### ISSUE TITLE
Type errors when building app with v5.3.1

### ISSUE THREAD

**Issue by will-hart** [pixijs/pixi.js#6774]
# Expected Behavior

Application builds without errors

### Current Behavior

Application build fails with type errors in `5.3.1`.

Builds successfully with `5.3.0`

### Possible Solution

Update function type signature in `pixi.js.d.ts`

### Steps to Reproduce

Install `5.3.1`, run a typescript-based build script. 

```
 ERROR  Failed to compile with 1 errors 
 error  in .../node_modules/pixi.js/pixi.js.d.ts

ERROR in .../node_modules/pixi.js/pixi.js.d.ts(4925,68):
4925:68 A required parameter cannot follow an optional parameter.
    4923 |              * @param {boolean} root - If is root
    4924 |              */
  > 4925 |             calculateProjection(destinationFrame?: PIXI.Rectangle, sourceFrame: PIXI.Rectangle, resolution: number, root: boolean): void;
         |                                                                    ^
    4926 |             /**
    4927 |              * Sets the transform of the active render target to the given matrix
    4928 |              *
```

### Environment

- **`pixi.js` version**: 5.3.1
- **Browser & Version**: N/A, fails in build process. Typescript 3.9.7
- **OS & Version**: Windows 10
- **Running Example**: N/A

**Comment by bigtimebuddy** [comment]
@SukantPal the optional JSDoc params here are incorrect:

https://github.com/pixijs/pixi.js/blob/c77bc2d5232e1fac1687b70c88b832b058f1eeae/packages/core/src/projection/ProjectionSystem.ts#L105-L108

**Comment by bigtimebuddy** [comment]
Merged and fixed in v5.3.2
